apps/cli/src/handleLoginCliCommand.ts

```diff
@@ -7,27 +8,54 @@ import { boldText, colorizeText, openURL } from "./utils.js";
 
 const ACCESS_TOKEN_REQUESTED_BY_CLI_KEY = "accessTokenRequestedByCLI";
 
+const routeUserToStudioForLogin = (printer: PrinterBlueprint) => {
+	printer.printConsoleMessage(
+		"info",
+		colorizeText(
+			"Opening the Codemod Studio... Please Sign in there!\n",
+			"cyan",
+		),
+	);
+	const success = openURL(
+		`https://codemod.studio/?command=${ACCESS_TOKEN_REQUESTED_BY_CLI_KEY}`,
+	);
+	if (!success) {
+		printer.printOperationMessage({
+			kind: "error",
+			message: "Unexpected error occurred while opening the Codemod Studio.",
```

Comment:
Avoid the term studio. 
"Unexpected error occurred while redirecting to sign-in page. Please submit a GitHub issue (sub.co shortlink to github issue template, so we can change it later via dub) or report to codemod.com/community"

Reply:
i think devs will feel more safe knowing that the URL links to github, so I will just leave it as native link